test/browser/Gemfile

```diff
@@ -0,0 +1,11 @@
+source 'https://rubygems.org'
+
+gem 'cocoapods'
```

Comment:
I don't think this needs cocoapods — isn't that just for the RN CLI tests?

Reply:
Yep, copy/paste error.